File: 프로그래머스/lv0/120956. 옹알이 （1）/옹알이 （1）.swift
import Foundation

func solution(_ babbling:[String]) -> Int {
    return babbling.filter {
        var str = $0
        ["aya", "ye", "woo", "ma"].forEach {
            str = str.replacingOccurrences(of: $0, with: "@")
        }
        return str.allSatisfy { $0 == "@" }
    }
    .count
}

File: 프로그래머스/unrated/120956. 옹알이 （1）/옹알이 （1）.java
class Solution {
    public int solution(String[] babbling) {
        int count = 0; 
        String[] syllables = {"aya", "ye", "woo", "ma"};

        for (String word : babbling) {
            String tempWord = word;
            for (String syllable : syllables) {
                if (tempWord.contains(syllable)) {
                    tempWord = tempWord.replace(syllable, "@");
                }
            }
            if (tempWord.replaceAll("@", "").isEmpty()) {
                count++;
            }
        }
        return count;
    }
}